src/index.html

```diff
@@ -2,11 +2,52 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+    rel="stylesheet"
+    href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+
+      <div class="calendar calendar__month--length--29">
+        <div class="calendar__day day day--1 calendar__start-day--mon"></div>
+        <div class="calendar__day day day--2"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```
remove `day` because it's an BEM-element, not BEM-block
remove `day--2` modificator - you don't need it - you should use `:nth-child` in the CSS to add correct styles instead of the modificator